src/overlays/actors/ovl_Door_Killer/z_door_killer.c

```diff
@@ -1,16 +1,35 @@
+/*
+ * File: z_door_killer.c
+ * Overlay: ovl_Door_Killer
+ * Description: Fake doors which attack player
+ *
+ * Associated switch flags: (params >> 8) & 0x3F
+ * ((params >> 8) & 0x3F) == 0x3F means no switch flags are checked / set
+ */
```

Comment:
you can put this near the relevant enums in the header (if there are any) since thats where people will go to find this information. at least until macros for it are created

Reply:
I'll move this to the header but I'm not sure I know how to write this kind of check in a macro without it being considered hiding too much info.

Would this be ok?
```
#define DOOR_KILLER_SWITCH_FLAG(params) (params >> 8) & 0x3F
...
if (DOOR_KILLER_SWITCH_FLAG(this->actor.params) != 0x3F) {
    Flags_SetSwitch(globalCtx, DOOR_KILLER_SWITCH_FLAG(this->actor.params))
}
```

Reply:
oh no no I said *until* macros are created. we havent worked out exactly how we want to write our macros that unpack complex params just yet. So just including a comment in the header that explains how they are packed is fine for now